audiorecorder/src/main/AndroidManifest.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?>
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+    package="org.odk.collect.audiorecorder">
+
+    <application>
+        <activity
+            android:name=".recording.AudioRecorderActivity"
+            android:label="Audio Recorder"
+            android:screenOrientation="portrait"
```

Comment:
Let's deal with rotation later as, once we have a Foreground Service responsible for recording, it won't be as big a deal.